wrappers/wrappers/deeptools/bamCoverage/wrapper.py

```diff
@@ -0,0 +1,29 @@
+import os
+from snakemake.shell import shell
+from lcdblib.snakemake import aligners
```

Comment:
don't think we need this; lcdblib's not in the environment.yaml. (I guess we miss stuff like this during testing when not testing wrappers with --use-conda)